src/libsyntax/feature_gate.rs

```diff
@@ -455,6 +455,9 @@ declare_features! (
 
     // Parentheses in patterns
     (active, pattern_parentheses, "1.26.0", None, None),
+
+    // The #[wasm_custom_section] attribute
+    (active, wasm_custom_section, "1.26.0", None, None),
```

Comment:
(don't we need a feature-gate test for this ?)

Reply:
Heh probably!